src/DataViews/SamplingReportDataView.cpp

```diff
@@ -61,6 +62,21 @@ const std::vector<DataView::Column>& SamplingReportDataView::GetColumns() {
   return columns;
 }
 
+[[nodiscard]] float SamplingReportDataView::HalfWidthOfSymmetrizedConfidenceInterval(
```

Comment:
I am still a bit unsure about this function, and it belongs to this class. Maybe it should be a function of the Estimator? Or of the Interval class itself? That would be probably also cleaner to test this.

At least it could be in some utils (worst case a namespace here), where the interval is passed to it along with the rate/percentage.

Reply:
You could use an anonymous namespace. But I also don't feel strong about this. I would be interested how @dpallotti and @beckerhe feel about this.

Reply:
I don't really have a clear opinion.
- The function does seem more general than the context of this class, and likely to be reused. For example, as soon as the confidence interval is applied to some other part of the UI, this code has to be either duplicated or move somewhere else.
- I can see how it doesn't really belong in `BinomialConfidenceInterval`/`Estimator` as it's more for visualization.

Maybe it could be a free function in a util file in `Statistics`?

Reply:
> Maybe it could be a free function in a util file in `Statistics`?
I thinks, it's the best option.